conditional_token_lists/conditional-token-list-chain-4.json

```diff
@@ -0,0 +1,682 @@
+{
```

Comment:
Should we version this folder?

Reply:
I think yes, for the moment, the requirement says `The list could perhaps be hosted in this repo or on IPFS.`